kgforge/specializations/stores/nexus/service.py

```diff
@@ -210,6 +212,12 @@ def make_endpoint(endpoint: str, endpoint_type: str, organisation: str, project:
             (endpoint, endpoint_type, quote_plus(organisation), quote_plus(project))
         )
 
+    @staticmethod
+    def add_resource_id_to_endpoint(
+            endpoint: str, resource_id: Optional[str]
+    ):
+        return "/".join([endpoint, quote_plus(resource_id)])
```

Comment:
Since `resource_id` is Optional, don't you need to check that it's different from `None` before using it?

Reply:
oh yes, missed that, fixed in the last commit